addon/services/intl.js

```diff
@@ -84,15 +84,19 @@ export default Service.extend(Evented, {
   /** @private **/
   _timer: null,
 
+  /** @private **/
+  _formats: null,
+
   /** @private **/
   _formatters: null,
 
-  _intls: null,
+  /** @private */
+  _intls: {},
 
   _cache: createIntlCache(),
 
   /** @public **/
-  init() {
+  init(opts) {
```

Comment:
Is this `opts` object being provided anywhere?  Cannot find where in this PR.  If not, I'd remove since I don't know of Ember services that do this currently.